Assets/MRTK/SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -323,6 +351,22 @@ public override void OnPostSceneQuery()
             }
         }
 
+        //public void Reset()
```

Comment:
I see this was commented out because of the potential removal - it might be good to look into how to make use of this code, or at least the parts that will raise a teleport cancelled when its disabled.